### DISCUSSION TITLE
DietPi-Software | NextDNS CLI

### DISCUSSION THREAD

**Discussion by rubeo-O** [MichaIng/DietPi#5615] [Software Request]
#### Formal software information
- Software title | NextDNS CLI
- Short description | Personal cloud and data sharing solution
- Official URL (if available) | https://nextdns.io
- GitHub URL (if available) | https://github.com/nextdns/nextdns
- (Official) install documentation | https://github.com/nextdns/nextdns/wiki

#### Are there similar/alternative software titles available with DietPi-Software?
Adguard Home

#### What makes your requested software better than the above solutions, if available?
NextDNS CLI is a DNS53 to DNS-over-HTTPS (DoH) proxy with advanced capabilities to get the most out of the NextDNS service. The CLI is fully open-source.

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
There are a few configuration options and for some the documentation is not exactly clear. DietPi could simplify the installation and configuration for less experienced Linux users.

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
Sorry, I am totally inexperienced in bash coding.

**Comment by Joulinar** [comment]
I moved the request to GitHub discussions. Vote for this software implementation with the ⬆️ button on this post.
